pkg/tests/support/helpers.py

```diff
@@ -1462,7 +1462,7 @@ def __attrs_post_init__(self):
         DaemonPkgMixin.__attrs_post_init__(self)
 
     def _get_impl_class(self):
-        if self.system_install and self.salt_pkg_install.system_service:
+        if self.system_service and self.salt_pkg_install.system_service:
```

Comment:
```suggestion
        if self.system_service:
```

I think we can change all these lines to this, correct @s0undt3ch